Source/WebInspectorUI/UserInterface/Views/Variables.css

```
@@ -354,100 +240,115 @@ body:is(.mac-platform.monterey, .mac-platform.big-sur):not(.docked) {
-
         --search-highlight-text-color: hsl(60, 100%, 50%);
         --search-highlight-text-color-active: hsl(0, 0%, 0%);
         --search-highlight-background-color: hsla(53, 83%, 53%, 0.2);
         --search-highlight-background-color-active: hsl(60, 100%, 50%);
         --search-highlight-underline-color: hsla(60, 100%, 50%, 0.5);
-
         --timeline-even-background-color: hsl(0, 0%, 25%);
-
         --timeline-scanner-color: hsl(0, 0%, 80%);
-
         /* TODO: Use the same variable for the default theme */
         --overlay-background: hsla(0, 0%, 24%, 0.9);
-
         --console-message-separator: var(--text-color-quaternary);
-
         --diff-addition-text-color: hsl(70, 64%, 70%);
         --diff-addition-background-color: hsl(90, 40%, 19%);
         --diff-deletion-text-color: hsl(0, 84%, 75%);
         --diff-deletion-background-color: hsl(5, 40%, 28%);
-
         --slider-thumb-background: hsl(0, 0%, 60%);
         --slider-thumb-background-active: hsl(0, 0%, 70%);
         --slider-thumb-box-shadow: 0px 0 2px 0 hsl(0, 0%, 10%);
         --slider-track-background: hsl(0, 0%, 35%);
         --slider-track-box-shadow: inset 0 0 3px hsl(0, 0%, 20%);
         --slider-track-tick-background: hsl(0, 0%, 55%);
     }
+}
+body {
+    &.window-inactive {
+        /* FIXME: Use CSS4 color blend functions once they're available. */
+        --selected-background-color: hsla(212, 92%, 54%, 0.5);
+        --glyph-color: hsla(0, 0%, 30%, 0.65);
+        --glyph-color-disabled: hsla(0, 0%, 30%, 0.35);
+        --glyph-opacity: 0.6;
+        --border-color: hsl(0, 0%, 85%);
+        @media (prefers-color-scheme: dark) {
+            --border-color: hsl(0, 0%, 33%);
+            --selected-background-color: hsla(212, 92%, 64%, 0.5);
+
+            --glyph-color: hsl(0, 0%, 52%);
+            --glyph-color-disabled: hsla(0, 0%, 52%, 0.4);
+        }
+    }
+    &:is(.mac-platform.monterey, .mac-platform.big-sur) {
+        --text-color: hsl(0, 0%, 15%);
+        --text-color-transparent: hsla(0, 0%, 15%, 0.4);
 
+        --background-color-intermediate: hsl(0, 0%, 98.5%);
 
-    body.window-inactive {
-        --border-color: hsl(0, 0%, 33%);
+        --panel-background-color: hsl(0, 0%, 96%);
+        --panel-background-color-light: hsl(0, 0%, 98%);
 
-        --selected-background-color: hsla(212, 92%, 64%, 0.5);
+        --separator-color: hsla(0, 0%, var(--foreground-lightness), 0.1);
 
-        --glyph-color: hsl(0, 0%, 52%);
-        --glyph-color-disabled: hsla(0, 0%, 52%, 0.4);
-    }
+        --border-color: hsl(0, 0%, 87%);
+        --border-color-secondary: hsl(0, 0%, 91%);
 
-    body:is(.mac-platform.monterey, .mac-platform.big-sur) {
-        --text-color: hsl(0, 0%, 85%);
-        --text-color-transparent: hsla(0, 0%, 85%, 0.4);
+        --sorted-header-font-weight: 700;
+        @media (prefers-color-scheme: dark) {
```

```suggestion
        --sorted-header-font-weight: 700;

        @media (prefers-color-scheme: dark) {
```
nit: newline for readability